CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to Nix
+
+A big welcome and thank you for considering contributing to Nix! It's people like you that make it a reality for users in our community.
+
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved.
+It also communicates that you agree to respect the time of the developers managing and developing these open source projects.
+In return, we will reciprocate that respect by addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+## Code of conduct
+
+We take our open source community seriously and hold ourselves and other contributors to high standards of communication.
+By participating and contributing to this project, you agree to uphold our [Code of conduct](https://www.contributor-covenant.org/version/2/1/code_of_conduct/).
+
+## Report a bug or suggest a feature
+
+1. Search whether your bug or feature was already reported. You can do so on [the GitHub issue tracker of nixos/nix](https://github.com/NixOS/nix/issues).
+
+2. If you were not able to find the bug or feature [open a new issue](https://github.com/NixOS/nix/issues/new/choose)
+
+3. The issue template will guide you in what should be in an issue.
```

Comment:
```suggestion
3. The issue templates will guide you in specifying your issue.
```

I think this is more specific and still brief. It's weird to say "something should be in an issue", at least it should be "issue description".